docs/reST/ref/scrap.rst

```diff
@@ -146,56 +149,68 @@ type.
    | :sl:`Places data into the clipboard.`
    | :sg:`put(type, data) -> None`
 
-   Places data for a specific clipboard type into the clipboard. The data must
-   be a string buffer. The type is a string identifying the type of data placed
-   into the clipboard. This can be one of the native ``SCRAP_PBM``,
-   ``SCRAP_PPM``, ``SCRAP_BMP`` or ``SCRAP_TEXT`` values or an own string
-   identifier.
+   Places data for a given clipboard type into the clipboard. The data must
+   be a string buffer. The type is a string identifying the type of data to be
+   placed into the clipboard. This can be one of the predefined
+   ``pygame.SCRAP_PBM``, ``pygame.SCRAP_PPM``, ``pygame.SCRAP_BMP`` or
+   ``pygame.SCRAP_TEXT`` values or a user defined string identifier.
 
-   The method raises an exception, if the content could not be placed into the
-   clipboard.
+   :param string type: type of data to be placed into the clipboard
+   :param data: data, of the given type, to be place into the clipboard
+   :type data: byte string (python 3) or str (python 2)
```

Comment:
I'm not sure how to do this type properly because it needs different ones with python 2 or 3. Various IDEs use the types, and I think they would need this to be `byte or str`? With the python2/3 note in the `:param data:` free form text.

In the (nearish) future we can (hopefully) move towards specifying the types in the '.pyi' type stubs (see buildconfig/pygame-stubs) and use https://github.com/agronholm/sphinx-autodoc-typehints to provide the types to sphinx. The issue for adding types is here: https://github.com/pygame/pygame/issues/839

Reply:
Yes the type field should just be type information. I've made the changes to fix this.